Lib/dis.py

```diff
@@ -32,7 +32,8 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, generators,
```

Comment:
I think this one is harder to get right in 72 characters, though (and as you point out below we can't just substitute a comment since it's public).

Technically correct but possibly confusing: `Disassemble classes, methods, functions, function-like objects, or code.`

`objects containing code objects` is technically correct(-ish), I think we need to mention functions explicitly in the top-level public docstring (unlike `_get_code_object`, where a mechanical definition may be clearer).

Is there a better generic phrase for generators, asynchronous generators, and coroutines? It seems wrong to mention generators explicitly but not the other two, but maybe that's the cost of living.